.vscode/tasks.json

```diff
@@ -53,11 +53,8 @@
     {
       "label": "Reset E2E tests",
       "type": "process",
-      "command": "docker-compose",
-      "args": ["restart", "app-for-e2e"],
-      "options": {
-        "cwd": "docker"
-      },
+      "command": "docker",
```

Comment:
not sure if this is viable but I'd like to see all "commands" run through make targets if possible, do you think that's possible here too?